src/common/darktable.c

```diff
@@ -103,6 +103,11 @@ darktable_t darktable;
 
 static int usage(const char *argv0)
 {
+#ifdef _WIN32
+  char *logdir = g_build_filename(g_get_user_cache_dir(), "darktable", NULL);
```

Comment:
What is `logdir` created for?

Reply:
To be able to print out later where the user can find the logfile, as its pretty hidden.

Reply:
I must be blind, I don't see it being used anywhere.

Reply:
Next line is using that `logdir`
`char *logfile = g_build_filename(*logdir*, "darktable-log.txt", NULL);`
Actually I have copied that from [here](https://github.com/darktable-org/darktable/blob/master/src/main.c#L43)
You mean maybe to use only one `g_build_filename()`?

Reply:
Yes. :-)

Reply:
This is also done with the latest commit